docs/site/src/docs/advanced/kyverno-build-in.md

```diff
@@ -2,21 +2,56 @@
 
 ## Motivations
 
-It's common that some k8s clusters have their own private image registry,and they don't want to pull images from other registry for safety reasons.this page is about how to introduce kyverno in k8s cluster,which will redirect image pull request to Specified registry.
+It's common that some k8s clusters have their own private image registry, and they don't want to pull images from other registry for safety reasons. This page is about how to introduce kyverno in k8s cluster, which will redirect image pull request to Specified registry.
```

Comment:
introduce -> integrate

`for safety reasons` not only for safety reasons, offline delivery also needs this feature.